src/modeler/AutoRest.Swagger/Validation/Core/ValidationChangesImpact.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace AutoRest.Swagger.Validation.Core
+{
+    [Flags]
+    public enum ValidationChangesImpact
+    {
+        None = 1 << 0,
+        PotentialServiceBreakingChanges = 1 << 1,
+        NonBreakingServiceChanges = 1 << 2
```

Comment:
should we call it NoServiceChangesRequired? or SdkImpacting or PotentialSdkChanges? Thinking about this, it seems like every property should have this property set, so a rule can impact the Service, or the SDK or nothing - but I wouldn't expect any rule to have no impact (as there wouldn't be any benefit on having the rule). 
Another option could be to name them "ServiceLevelImpacting"/"SdkLevelImpacting" or just "ServiceImpact"/"SdkImpact". 
@kirthik thoughts?